jena/src/test/java/org/apache/commons/rdf/jena/TestJenaGraphToCommonsRDFGraph.java

```diff
@@ -33,7 +32,7 @@
 import org.apache.commons.rdf.api.RDFTerm;
 import org.apache.commons.rdf.api.Triple;
 import org.apache.commons.rdf.simple.Types;
-import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.Lang ;
```

Comment:
Well spotted! I think it came from @afs's original code  - I have removed the spaces.